src/components/MarkdownPlugins/image.js

```diff
@@ -7,9 +7,9 @@ const image = {
     image: match[2],
     alt: match[1],
   },
-  toBlock: data => `![${ data.alt || '' }](${ data.image || '' })`,
+  toBlock: data => `![${ data.alt }](${ data.image || '' })`,
   toPreview: (data, getAsset) => <img src={getAsset(data.image || '')} alt={data.alt || ''} />,
```

Comment:
@erquhart I noticed that you modified this line as well in your 1.0 changes -- I had it as `src={getAsset(data.image) || ''}`.

Reply:
@tech4him1 good catch, yes, that was unintentional.